dotcom-rendering/src/web/experiments/tests/sign-in-gate-main-variant.ts

```diff
@@ -19,7 +19,15 @@ export const signInGateMainVariant: ABTest = {
 	canRun: () => true,
 	variants: [
 		{
-			id: 'main-variant-4',
```

Comment:
TODO -> this should be reverted back to `main-variant-4`. 
It's like this for now because I couldn't quite figure out how to configure the audience segmentation buckets.